pdns/dnsrecords.hh

```diff
@@ -786,6 +786,28 @@ private:
  uint8_t d_eui64[8];
 };
 
+#define APL_FAMILY_IPV4 1
+#define APL_FAMILY_IPV6 2
+class APLRecordContent : public DNSRecordContent
```

Comment:
why not just use 

`includeboilerplate(APL)` here?

Reply:
I'm new to the pdns code base and I copied the structure of the EUI64RecordContent RR for the APL RR. 
I'll switch to the boilerplate macro and retest.